vars/provision.md

```diff
@@ -0,0 +1,25 @@
+# provision
+
```

Comment:
We should include a brief summary of what the global var is used for like we have with the others.

ie.

`> Dynamically provision cloud resources`